resource-management/pkg/aggregrator/aggregator.go

```diff
@@ -136,6 +136,16 @@ func (a *Aggregator) Run() (err error) {
 						a.postCRV(c, crv)
 					}
 				}
+
+				endPullAndProcess := time.Now()
+				completeDuration := endPullAndProcess.Sub(startPullAndProcess)
+
+				// For performance increasing, change interval from 100ms to 10ms, even less than 10ms
+				completeDurationMillisecondsCount := completeDuration.Milliseconds()
```

Comment:
Line 144-147 can be simplified to:
if defaultPullInterval > completeDuration {
    time.Sleep(defaultPullInterval - completeDuration)
}

Reply:
defaultPullInterval is 10 default, completeDuration is "time.Duration" type. they should not be directly compared each other.

I use VScode IDE to verify simplified codes are not working.

The below codes are working. Actually there is no big difference from my previous codes

                                // Get how many milliseconds of latency
                                completeDuration := endPullAndProcess.Sub(startPullAndProcess).Milliseconds()

                                // For performance increasing, change interval from 100ms to 10ms, even less than 10ms
                                if defaultPullInterval > completeDuration {
                                        time.Sleep(time.Duration(defaultPullInterval - completeDuration) * time.Millisecond)
                                }

Reply:
My point is, there is no necessary to get milliseconds out of completeDuration, DefaultPullInterval should be initialized as duration, which is 10 * Millisecond. No need to convert the value multiple times.

Reply:
get your point